.gitignore

```diff
@@ -0,0 +1,22 @@
+.idea/
+.idea/*
+
+lib/
+lib/*
+
+*.py[cod]
+*.pyc
+
+bin/
+bin/*
+
+share/
+share/*
+
+local/
+local/*
+
+include/
+include/*
```

Comment:
Are you sure you have to duplicate the folder names? Wouldn't be the lines with asterisk at the end enough?